### ISSUE TITLE
Hubot is not answering to commands.

### ISSUE THREAD

**Issue by patsToms** [benjamine/hubot-skype#1]
Hi, I had the same problem with hubot-skype package from npm by netpro2k.

When I setup module from your repo I saw that adapter is getting my messages.
I post some log output here - http://sprunge.us/KUGL

Any ideas why it's not working?
I am using Debian i386 with node v0.8.20.

Here is the npm list.
├── coffee-script@1.4.0
├─┬ connect@2.3.4
│ ├── bytes@0.0.1
│ ├── cookie@0.0.4
│ ├── crc@0.2.0
│ ├── debug@0.7.2
│ ├── formidable@1.0.11
│ ├── fresh@0.0.1
│ ├── mime@1.2.4
│ ├── qs@0.4.2
│ └── range-parser@0.0.4
├── connect_router@1.8.6
├── hubot@2.4.6
├── hubot-skype@0.0.3
├── log@1.3.0
├── optparse@1.0.3
└── scoped-http-client@0.9.7

thanks in advance.

**Comment by wpK** [comment]
@patsToms

I had the same issue as you. What fixed it for me was to create a deployable hubot (hubot -c PATH) and then configure the package.json with hubot-skype.

**Comment by benjamine** [comment]
Hi guys, sorry for the delay, thanks @wpK , what you say almost confirms my theory, I had similar issues in the past,
the problem I think is I declared hubot as a dependency to hubot-skype, and that, depending on how you installed it, can cause you to get hubot installed twice, then hubot-skype talks to a different hubot API making messages get lost.
If that's the case, removing hubot folder from:

```
/yourhubotfolder/node_modules/hubot-skype/node_modules/hubot
```

and letting only the one at:

```
/yourhubotfolder/node_modules/hubot
```

should fix it, I'll remove hubot from package.json dependencies later.

**Comment by patsToms** [comment]
Thanks about answers. @wpK, what do you mean with deployable? Can you explain more please? 

I have random hubot which I run with bin/hubot -a skype as said in hubot documentation.

**Comment by wpK** [comment]
@benjamine That makes sense, since this was the first time I've used NodeJS I spent a lot of time yesterday trying to figure out the issue. I ran hubot with the Skype adapter through node-inspector and it was really confusing on why it was trying to match the message with the listener regex's, and even though it would match it would never call the listener correctly.

@patsToms Try what @benjamine posted and see if that fixes the issue. If it doesn't, try the following:

```
run bin/hubot -c ../hubot-deploy
cd ../hubot-deploy
```

Then in this new directory, edit packages.json to add the skype-adapter and then install the modules with the npm install. Basically you're going to setup hubot the exact same way you did originally, but in hubot-deploy.
